src/types/WalletConnectService.ts

```diff
@@ -7,10 +7,12 @@ import type { GasPrice } from './GasPriceType';
 
 export type WalletConnectClientType = WalletConnectClient;
 
-type WCSPendingMethod = '' | 'sendMessage' | 'signJWT' | 'signHexMessage' | 'sendWalletAction';
+type WCSPendingMethod = '' | 'sendMessage' | 'signJWT' | 'signHexMessage' | 'switchToGroup';
 
 export type WalletConnectServiceStatus = 'connected' | 'disconnected' | 'pending';
 
+export type WalletAction = 'switchToGroup';
```

Comment:
This is already a const in src/consts/walletActions.ts.
Change it to be 
```export type WalletAction = 'switchToGroup' as const;``` within walletActions.ts and you can import it as the type as well.